bears/natural_language/WriteGoodLintBear.py

```diff
@@ -0,0 +1,67 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bears.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='write-good',
+        output_format='regex',
+        output_regex=r'(?P<message>.*)\s*on\s*line\s*(?P<line>\d+)\s*at\scolumn'
+                      '\s*(?P<column>\d+)'
+        )
+class WriteGoodLintBear:
+    """
+    Lints the text files using ``write-good`` for improving proses.
+    For more information on this tool, visit
+    https://github.com/btford/write-good
```

Comment:
i think you should be
``See <https://github.com/btford/write-good>  for more information.``
this is just for consistency across the bears :)